test/suites/container_devices_nic_physical.sh

```diff
@@ -38,7 +38,7 @@ test_container_devices_nic_physical() {
   fi
 
   # Stop container and check MTU is restored.
-  lxc stop -f "${ctName}"
+  lxc stop "${ctName}"
```

Comment:
All calls to `lxc stop` and `lxc restart` must pass `-f` when called from the testsuite.
We have some test environments where busybox doesn't properly handle signals, causing tests to hang.

Reply:
@stgraber cool, I've applied that to a few more cases too.